extensions/multiline.md

```diff
@@ -80,7 +80,7 @@ Servers MUST NOT reject blank lines other than in the following cases:
 * Clients MUST NOT send blank lines with the `draft/multiline-concat` tag.
 * Clients MUST NOT send messages consisting entirely of blank lines.
 
-Clients MUST NOT send messages other than PRIVMSG while a multiline batch is open.
+Once the client has opened a multiline batch, it MUST NOT send any messages that are not part of the batch, i.e., it may only send PRIVMSG or NOTICE with the appropriate `batch` tag, or the closing `BATCH` command.
```

Comment:
`it may only send PRIVMSG or NOTICE` -> `it may send only PRIVMSG or only NOTICE`

Reply:
I really think that's implicit in the previous change?

Reply:
It is, but it doesn't hurt to say it again. No big deal either way